pixel-point/pixelpoint-website
# feat: update marketing section

**Describe what changes this pull request brings**
Updates integrations section

[Design](https://www.figma.com/file/uRyIVXOk6qD7ufNv6JSQRf/Pixel-Point-Website-Design?type=design&node-id=6298-2348&mode=design&t=rppPVN9uYhnznaCH-4)

**Steps to test**

Open the [page](https://pixelpoint-website-git-update-marketing-section-pixelpoint.vercel.app/services/web-design/)
Confirm that everything looks and works as expected.
